src/private/files.ts

```diff
@@ -0,0 +1,35 @@
+import { ensureInitialized, sendMessageRequest } from "../internal/internalAPIs";
+import { GlobalVars } from "../internal/globalVars";
+
+/**
+ * Namespace to interact with the files part of the SDK.
+ * This object is used to handle logs
+ *
+ * @private
+ * Hide from docs
+ */
+export namespace files {
```

Comment:
please rename this file and namespace to more logical with logs ? May be AppLogs @ydogandjiev  what you think?

Reply:
Probably `log` or `logs` would be best and that's kind of the name we've chosen by using `log.request` as the message name.

Reply:
renamed to logs